ena_upload/templates/ENA_template_studies.xml

```diff
@@ -8,16 +16,29 @@
         <DESCRIPTOR>
             <STUDY_TITLE>${row.title}</STUDY_TITLE>
             <STUDY_TYPE existing_study_type="${row.study_type}"/>
+            <py:if test="attributetest(row, 'study_abstract')">
             <STUDY_ABSTRACT>${row.study_abstract}</STUDY_ABSTRACT>
+            </py:if>
+            <py:if test="attributetest(row, 'center_name')">
+            <CENTER_NAME>${row.center_name}</CENTER_NAME>
```

Comment:
While the SRA.study.xsd has `center_name` as optional, the [ENA documentation](https://ena-docs.readthedocs.io/en/latest/submit/general-guide/programmatic.html#identifying-submitters) describes it as mandatory for broker accounts.
Might be easier to keep this one as mandatory since the tool is also supposed to be used by brokers. In which case, adding a mention of having this as a mandatory field in the documentation might be a good addition.

Reply:
I I think this is still the case, but if I understand the XSD correctly it is also a possible field within the DESCRIPTOR

Reply:
Indeed, there is a `center_name` inside DESCRIPTOR, but I can't find examples of its use there by manually browsing through studies in ENA. They seem to be having `center_name` directly at Study, together with `accession` and `alias`.

Reply:
should I eave it out than to take away confusion?

Reply:
I agree with leaving out the `center_name` that lives inside of DESCRIPTOR.
As I understand, the `CENTER_NAME` argument of the ENA-upload-cli already takes care of this field in a way that makes more sense to ENA.